tests/Amazon/SNS/Verify/ValidURISpec.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE NoOverloadedStrings #-}
+
+module Amazon.SNS.Verify.ValidURISpec
+  ( spec
+  ) where
+
+import Amazon.SNS.Verify.TestPrelude
+
+import Amazon.SNS.Verify.ValidURI
+import Text.Regex.TDFA ((=~))
+
+spec :: Spec
+spec = around_ useCertServer $ do
+  describe "verifySNSMessage" $ do
+    it "validates a prod schema" $ do
+      "sns.us-east-2.amazonaws.com" =~ prodRegPattern `shouldBe` True
```

Comment:
FWIW, this will give you a pretty un-useful failure ("Expected `True`, go `False`). If you use something like,

```hs
"sns.us-east-2.amazonaws.com" `shouldSatisfy` (=~ prodRegPattern)
```

It should report the failing value.

Not a big deal here I suppose, but I avoid `` `shouldBe` True`` for this reason generally.